### PR TITLE
Make Elevator

### PR DESCRIPTION
**Pull request by connorgannaway**
Create an elevator subsystem and command. When "select" is pressed, it toggles the motor for the elevator on/off.

### CODE REVIEW

File: src/main/java/frc/robot/Constants.java [hunk 1]
```diff
@@ -19,6 +19,9 @@
     //controller port
     public static final int controllerport = 0;
     //motor controller ports
+    public static final int elevatorPort = 4;
+    //other constants
+    public static final double elevatorSpeed = 0.6; //speed for elevator to move at
     public static final int leftmotor = 0;
     public static final int rightmotor = 1;
     public static final int outputMotorPort = 2;
```
Comment (JacobMalloy) [inline src/main/java/frc/robot/Constants.java:25]: I don't know if this is your fault, but these should be under the motor controller ports, not other constants.

File: src/main/java/frc/robot/subsystems/Elevator.java [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+import com.ctre.phoenix.motorcontrol.can.VictorSPX;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+
+public class Elevator extends SubsystemBase {
+
+  //creates instance of victorspx motor controller for elevator motor
+  VictorSPX motor = new VictorSPX(Constants.elevatorPort);
+
```
Comment (JacobMalloy) [inline src/main/java/frc/robot/subsystems/Elevator.java:20]: Could I have a brief comment explaining what this variable does?

File: src/main/java/frc/robot/subsystems/Elevator.java [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+   * Toggles "toggle" to activate onSwitch & offSwitch
+   */
+  public void toggler(){
+    toggle ^= true;
+    if(toggle == true){
+      onSwitch();
+    }
+    else{
+      offSwitch();
+    }
```
Comment (JacobMalloy) [inline src/main/java/frc/robot/subsystems/Elevator.java]: I need a couple of comments over this section as well.

**Review by JacobMalloy — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/subsystems/Elevator.java (hunk 2 above)
Comment (connorgannaway) [reply src/main/java/frc/robot/subsystems/Elevator.java:20]: redid the comment

Comment (JacobMalloy) [reply src/main/java/frc/robot/subsystems/Elevator.java:20]: I meant the toggle boolean. the comment on the motor was fine.

**Review by JacobMalloy — CHANGES_REQUESTED** [review]

**Review by JacobMalloy — APPROVED** [review]

**Pull request merged**